x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/risk_engine/risk_engine_privileges.ts

```diff
@@ -97,7 +97,10 @@ export const withRiskEnginePrivilegeCheck = <P, Q, B>(
   ) => {
     const [_, { security }] = await getStartServices();
     const privileges = await getUserRiskEnginePrivileges(request, security);
-    if (!privileges.has_all_required) {
+    if (
+      !privileges.has_all_required &&
+      !privileges.privileges.elasticsearch.cluster?.manage_transform
```

Comment:
Why are you checking again for `manage_transform`? As far as I know, it has already been checked inside `getUserRiskEnginePrivileges`.